js/client-teambuilder.js

```diff
@@ -1200,6 +1200,7 @@
 					}
 				}
 				if (this.curTeam.gen === 8) {
+					if (!species.cannotDynamax) buf += '<span class="detailcell"><label>Dmax Level</label>' + (typeof set.dynamaxLevel === 'number' ? set.dynamaxLevel : 10) + '</span>';
```

Comment:
we probably shouldnt display this and should arguably remove more things because it causes the UI to become extremely ugly for people with smaller screens

Reply:
I definitely think it should be left off if it's 10.

Reply:
I think it makes sense to leave it in if it's some other number, though.